source/contributed/ps_ubuntu.md

```diff
@@ -1,25 +1,23 @@
 ---
 title: Private server on Ubuntu using MongoDB and Redis
 contributed:
-    name: tedivm
-    link: https://github.com/tedivm
-    date: 2018-02-07
+  name: tedivm
+  link: https://github.com/tedivm
+  date: 2018-02-07
```

Comment:
This formatting has meaning- this blob is essentially yaml, so when you remove those spaces you're turning these variables from being underneath the `contributed` key to being top level variables.

Reply:
thanks. Will fix and get out the no spray for VS Code

Reply:
It looks like this is just tabs being replaced by spaces. They are still indented two times in and is valid YAML:
```
title: Private server on Ubuntu using MongoDB and Redis
contributed:
  name: tedivm
  link: https://github.com/tedivm
  date: 2018-02-07
```